scheme/data.py

```diff
@@ -0,0 +1,153 @@
+from faker import Faker
+from decimal import Decimal
+import random as r
+import json
+
+
+f = Faker()
+Faker.seed(0)  # we are using the same data all the time here
+r.seed(0)
+lat = 37.773972
+lng = -122.431297
+
+
+def genId():
+    return f.md5()
+
+
+def pickOne(lst):
+    return lst[r.randint(0, len(lst)-1)]
```

Comment:
Minor: more readable version: return random.choice(lst)

Reply:
nice, ehm, forgot about this entirely